code/obj/cryotron.dm

```diff
@@ -318,29 +318,31 @@
 					if (!isnull(crew_record))
 						crew_record["p_stat"] = "Active"
 
-	attack_hand(var/mob/user)
-		if(isgrab(user.l_hand))
+	/// Helper to check if a player is trying to insert another player
+	proc/cryo_attack_hand(var/mob/user)
+		if (isgrab(user.l_hand))
 			src.Attackby(user.l_hand, user)
-		else if(isgrab(user.r_hand))
+		else if (isgrab(user.r_hand))
 			src.Attackby(user.r_hand, user)
-		else if (!enter_prompt(user))
-			return ..()
```

Comment:
I don't think you can just replace `attack_hand` like this. Why not just put it in the regular `attack_hand`? Why was a new proc needed?

Reply:
The reason I'm removing the attack_hand/attackby/attack_ai is because the yare now handled by Click. The Click implementation was done initially to allow critters without hands to cryo, but that cause the issue that now critters and mobs with hands would get the prompt twice, once in attack_by and once in Click. 

So the solution could be to either 1. Keep both and type filter in Click which is kinda nasty as it requires you to keep a list of types to filter out, or 2. Remove the implementations of attack_hand / attackby / attack_ai and route them through Click. The methods that remain are now local to cryotron (cryo_attack_hand, cryo_attackby). 

I kept them this way so the code would read a bit better, but I could inline the functionality to the caller (Click) if desired.

Reply:
Ah, I see what you mean. Inlining it into the Click proc may make it clearer, yes. The naming of those procs is a bit... ambiguous. At least to me.

Reply:
Done